### ISSUE TITLE
Google home/Google tv android remote apps do not work on Android TV

### ISSUE THREAD

**Issue by parkgrrr** [xbmc/xbmc#21535]
### Describe the bug

Google has deprecated the old android TV remote app, and has a new remote in both the Google home and Google tv apps. These apps do not work when launching Kodi, I assuming the necessary mapping is missing.

## Expected Behavior
User should be able to navigate inside Kodi just like other apps

## Actual Behavior

The navigation buttons do not work, and users are unable to navigate 

## Possible Fix

Include remote mapping for the new Google Home and Google TV remote apps

### To Reproduce

1. install Google home or Google tv to an android device
2. Turn on an Android TV device
3. Connect the remote app to Android TV
4. Launch Kodi and try to navigate 

### Debuglog

The debuglog can be found here:

### Screenshots
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [x] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name:
 - Kodi version: V.19.3 Matrix

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by DaVukovic** [comment]
Can you confirm being able to navigate inside Kodi using Yatse and/or Kore?

**Comment by enen92** [comment]
For some reason I can no longer reproduce this issue on my main TV. Please check if this is still relevant on systems with all Google services up to date.

**Comment by enen92** [comment]
Closing as I can no longer reproduce this.

**Comment by stbenz** [comment]
Just found this issue after playing around with the Android TV Remote integration of Home Assistant (uses the same protocol as the Google TV Remote).

I currently have the described problem, that I can't control Kodi from the Google TV Remote app on my phone and the Android TV Remote integration of Home Assistant. Using the physical remote works fine, as does the Yatse app on my phone.

My Android TV device is a Chromecast with Google TV 4K running the latest Kodi app (20.1). There aren't any updates available for the Chromecast (neither system updates nor app updates) and my phone.

All the other apps on the Chromecast work fine, except for the Netflix app, and it seems that other people have the same problem with the Netflix app.

Another thing I noticed: In the Android TV Remote integration of Home Assistant I can configure additional commands that are not available on the physical remote and the Google TV Remote app. I tried some media control commands (MEDIA_PREVIOUS, MEDIA_NEXT, MEDIA_STOP, MEDIA_PLAY_PAUSE) and they work with Kodi. So it seems that the virtual remote works at least partially, although all the available buttons in the Google TV Remote app don't work.
